pytorch_blade/src/ltc/disc_compiler/BUILD

```diff
@@ -1,20 +1,37 @@
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension", "pybind_library")
 
 cc_library (
-  name = "disc_compiler",
+  name = "disc_passes",
   srcs = [
-    "disc_compiler.cpp",
-    "passes/cluster.cpp",
+    "passes/disc_fuser.cpp",
     "passes/register_disc_class.cpp",
+    "passes/graph_fuser.cpp",
   ],
   hdrs = [
-    "disc_compiler.h",
     "passes/io.h",
-    "passes/cluster.h",
+    "passes/disc_fuser.h",
     "passes/register_disc_class.h",
+    "passes/graph_fuser.h",
+  ],
+  deps = [
+    "@org_torch_blade//src/compiler/mlir:torch_blade_mlir",
```

Comment:
That raised an error If only use `//src:torch_blade_shared `:

``` text
src/ltc/disc_compiler/passes/register_disc_class.cpp:12:10: fatal error: ltc/disc_compiler/passes/register_disc_class.h: No such file or directory
```

Actuall, we want `//src:torch_blade_shared ` depends on `torch_disc`, because we only need one `torch_blade` library.

Reply:
It's weird since `//src:torch_blade_shared` is not related to `ltc/disc_compiler/passes/register_disc_class.h`. To solve the error posted here you might need to add `//src:torch_blade_include` to `deps`.

And agree with you that we finally should let `torch_blade` depend on `torch_disc`. But perhaps we can refactor it later. The tricky issue here is that `@org_torch_blade//src/compiler/mlir:torch_blade_mlir` is already static linked in `libtorch_blad.so`.